monitoring/datadog-dashboard.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
@sgrebnov let's also add a `README.md` to the `monitoring` directory and point people at the Docs for how to use each dashboard.

Reply:
Confirm this will be added as part of next PR with Datadog dashboard improvements